cognitedata/reveal
# fix(react-components): unknown space `cog_3d_data` error when models are added into different space

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-4100

## Description :pencil:
If 3D models are not added into `cog_3d_data` space than on node clicking it would result in console error, this PR fixes the issue by fetching and maintain space the 3D model belong to.
 
## How has this been tested? :mag:

In `Search`


## Test instructions :information_source:

- `yarn && yarn build && yalc publish` in `react-components`
- `yalc add @cognite/reveal-react-components && yarn && nx serve flexible-data-explorer` in `fusion` repo
- Load `omv` project and browse to 3D page
- Load `Maui Production Station` site
- Click on any 3D node, console.error should not popup

## Checklist :ballot_box_with_check:
